docs/source/about/third_party_packages.rst

```diff
@@ -8,7 +8,7 @@ AOSC
    :alt: AOSC Version
-   :target: https://packages.aosc.io/packages/sunshine
+   :target: https://aosc-packages.cth451.me/packages/sunshine
```

Comment:
Probably just remove this entire item. It's no longer available here: https://repology.org/project/sunshine/versions

Which is where we were pulling the version from.

Reply:
I made another commit that removes AOSC. It is now completely removed from docs/source/about/third_party_packages.rst

Reply:
Thanks!